transformer_class.py

```diff
@@ -158,7 +59,7 @@ def __init__(self, **kwargs):
     def supported_image_file_exts(self):
         """Returns the list of supported image file extension strings (in lower case)
         """
-        return ['tif', 'tiff', 'jpg']
+        return Transformer.supported_file_exts
```

Comment:
This change should be reverted to what it was since it's Transformer specific. IMO having the `supported_file_exts` function in Transformer was a bad design decision I made that shouldn't get brought forward.